changelog/bugfixes/2024-03-22-toolbox-prevent-snapshots-from-being-gced.md

```diff
@@ -0,0 +1 @@
+- Updated toolbox which fixes that mounted snapshots with ctr are being gc'ed which breaks the toolbox.
```

Comment:
```suggestion
- Fixed `toolbox` to prevent mounted ctr snapshots from being garbage-collected ([toolbox#9](https://github.com/flatcar/toolbox/pull/9))
```